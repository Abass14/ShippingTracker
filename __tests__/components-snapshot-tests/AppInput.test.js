import ShallowRenderer from 'react-shallow-renderer';
import AppInput from '../../src/app/components/input';

it('should render AppInput correctly', () => {
    const renderer = new ShallowRenderer()
    const tree = renderer.render(
        <AppInput />
    )
    expect(tree).toMatchSnapshot()
})