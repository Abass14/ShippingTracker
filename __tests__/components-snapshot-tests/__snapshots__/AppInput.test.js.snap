// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AppInput correctly 1`] = `
<View
  style={
    {
      "gap": 4,
    }
  }
>
  <View
    style={
      [
        {
          "backgroundColor": "#F4F2F8",
          "borderColor": "#2F50C1",
          "borderRadius": 10,
          "gap": 4,
          "height": 56,
          "justifyContent": "center",
          "paddingHorizontal": 12,
          "paddingVertical": 6,
          "width": "100%",
        },
        null,
        null,
      ]
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "gap": 5,
          "width": "100%",
        }
      }
    >
      <TextInput
        onBlur={[Function]}
        onFocus={[Function]}
        style={
          [
            {
              "alignItems": "center",
              "color": "#2F50C1",
              "flex": 1,
              "fontSize": 14,
              "paddingVertical": 5,
            },
            undefined,
          ]
        }
      />
    </View>
  </View>
</View>
`;
